WordPress/Docs/PHP/YodaConditionsStandard.xml

```diff
@@ -0,0 +1,26 @@
+<documentation title="Yoda Conditional Statements">
+    <standard>
+    <![CDATA[
+      When doing logical comparisons involving variables, the variable must be placed on the right side. All constants, literals, and function calls must be placed on the left side. If neither side is a variable, the order is unimportant. 
+      
```

Comment:
Just noticed some trailing whitespace here.

Reply:
Still seems to be whitespace present here.

<img width="456" alt="Screenshot 2019-06-20 at 13 30 09" src="https://user-images.githubusercontent.com/88371/59846229-91df0880-935f-11e9-91e6-f0a80e66f645.png">